db/migrate/20200710101438_add_non_uk_gcse_columnns_to_application_qualification.rb

```diff
@@ -0,0 +1,8 @@
+class AddNonUkGcseColumnnsToApplicationQualification < ActiveRecord::Migration[6.0]
+  def change
+    add_column :application_qualifications, :non_uk_qualification_type, :string
+    add_column :application_qualifications, :accrediting_country, :string
+    add_column :application_qualifications, :naric_reference_number, :string
+    add_column :application_qualifications, :comparable_uk_qualification, :string
```

Comment:
We've already got a `naric_reference` and `comparable_uk_degree`. I wonder whether we should be using the same columns at least for the reference. Maybe I should drop `comparable_uk_degree` and use `comparable_uk_qualification` for everything? It could get very confusing otherwise.

Reply:
Yea I think it probably makes sense to use `comparable_uk_qualification` for all of the qualifications.